### PR TITLE
Blog post for Synapse 1.78

### PR DESCRIPTION
**Pull request by H-Shay**
Preview: https://pr1675--matrix-org-previews.netlify.app

### CODE REVIEW

File: gatsby/content/blog/2023/02/2023-02-28-synapse-1.78.mdx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+will be removed in a future release.
+
+## The new stuff
+
+In case you were unaware, Synapse has a command line export data command which allows administrators to export data for
+a specific user (you can read more about this command [here](https://matrix-org.github.io/synapse/latest/usage/administration/admin_faq.html#how-can-i-export-user-data)).
+Synapse 1.78.0 updates this command adding account data to the user information that is returned by the export data command.
+
+This version of Synapse also features a few changes to push rules, with implementations for [MSC3758](https://github.com/matrix-org/matrix-spec-proposals/pull/3758),
+[MSC3966](https://github.com/matrix-org/matrix-spec-proposals/pull/3966), and the removal of the spurious `dont_notify`
```
Comment (reivilibre) [inline gatsby/content/blog/2023/02/2023-02-28-synapse-1.78.mdx:27]: maybe include at least the title of the MSC, for lazy readers like me :)

File: gatsby/content/blog/2023/02/2023-02-28-synapse-1.78.mdx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+action from the defaults for the `.m.rule.reaction` pushrule.
```
Comment (reivilibre) [inline gatsby/content/blog/2023/02/2023-02-28-synapse-1.78.mdx:28]: can we possibly expand on what this means for users? I don't understand what it means to be honest and I thought I knew about push rules!

File: gatsby/content/blog/2023/02/2023-02-28-synapse-1.78.mdx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+Synapse v1.78.0 includes some fixes for faster-joins related bugs as we work out the kinks. Some of these include a fix
+for a bug introduced in Synapse 1.76.0 where partially-joined rooms could not be deleted using the
+[purge room API](https://matrix-org.github.io/synapse/latest/admin_api/rooms.html#delete-room-api), and
+a fix for a bug introduced in Synapse 1.75 where the [portdb script](https://matrix-org.github.io/synapse/latest/postgres.html#porting-from-sqlite)
+would fail to run after a room had been faster-joined. We continue to work to polish faster joins, and thank everyone who
+filed an issue.
+
```
Comment (reivilibre) [inline gatsby/content/blog/2023/02/2023-02-28-synapse-1.78.mdx:36]: I would be tempted to add an honorary mention https://github.com/matrix-org/synapse/issues/15150, which is a bug that would be biting workerised deployments.

File: gatsby/content/blog/2023/02/2023-02-28-synapse-1.78.mdx (hunk 2 above)
Comment (H-Shay) [reply gatsby/content/blog/2023/02/2023-02-28-synapse-1.78.mdx:28]: lol honestly I couldn't really entirely parse out what it meant for end users either but I needed to write about something...

File: gatsby/content/blog/2023/02/2023-02-28-synapse-1.78.mdx (hunk 3 above)
Comment (H-Shay) [reply gatsby/content/blog/2023/02/2023-02-28-synapse-1.78.mdx:36]: good shout will do

File: gatsby/content/blog/2023/02/2023-02-28-synapse-1.78.mdx [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+will be removed in a future release.
+
+## The new stuff
+
+In case you were unaware, Synapse has a command line export data command which allows administrators to export data for
+a specific user (you can read more about this command [here](https://matrix-org.github.io/synapse/latest/usage/administration/admin_faq.html#how-can-i-export-user-data)).
+Synapse 1.78.0 updates this command adding account data to the user information that is returned by the export data command.
+
+This version of Synapse also features a few changes to push rules, with implementations for [MSC3758](https://github.com/matrix-org/matrix-spec-proposals/pull/3758):
+Add event_property_is push rule condition kind,
```
Comment (anoadragon453) [inline gatsby/content/blog/2023/02/2023-02-28-synapse-1.78.mdx:27]:
```suggestion
Add `event_property_is` push rule condition kind,
```

File: gatsby/content/blog/2023/02/2023-02-28-synapse-1.78.mdx (hunk 2 above)
Comment (anoadragon453) [reply gatsby/content/blog/2023/02/2023-02-28-synapse-1.78.mdx:29]: Basically they are fundamental tools that give people greater flexibility in customising their notification rules. With MSC3758, you can now ask to be notified when a field in an event contains any given type of value. Before that, you could only match on strings.

And MSC3966 allows for sending a notification when an item contained in an array value in an event matches a given pattern. This is a change that's necessary for MSC3952 - specifically to see if your Matrix ID has been included in a new `m.mentions.user_ids` field on an event, which that MSC defines.

File: gatsby/content/blog/2023/02/2023-02-28-synapse-1.78.mdx [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+action from the defaults for the `.m.rule.reaction` pushrule.
+
+Synapse v1.78.0 includes some fixes for faster-joins related bugs as we work out the kinks. Some of these include a fix
+for a bug introduced in Synapse 1.76.0 where partially-joined rooms could not be deleted using the
+[purge room API](https://matrix-org.github.io/synapse/latest/admin_api/rooms.html#delete-room-api), and
+a fix for a bug introduced in Synapse 1.75 where the [portdb script](https://matrix-org.github.io/synapse/latest/postgres.html#porting-from-sqlite)
+would fail to run after a room had been faster-joined. We continue to work to polish faster joins, and thank everyone who
+filed an issue.
+
+Finally, for those deployments using workers v1.78.0 fixed a bug introduced in Synapse 1.76 where 5s delays would
```
Comment (anoadragon453) [inline gatsby/content/blog/2023/02/2023-02-28-synapse-1.78.mdx:38]:
```suggestion
Finally, for those deployments using workers, v1.78.0 fixed a bug introduced in Synapse 1.76 where 5s delays would
```

**Review by anoadragon453 — COMMENTED** [review]
A clear and interesting post - great job!

File: gatsby/content/blog/2023/02/2023-02-28-synapse-1.78.mdx (hunk 2 above)
Comment (H-Shay) [reply gatsby/content/blog/2023/02/2023-02-28-synapse-1.78.mdx:29]: Thanks for the insight anoa - I am going to nick some of this explanation for the blog :)

**Review by anoadragon453 — APPROVED** [review]

**Review by reivilibre — APPROVED** [review]

**Pull request merged**